programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1618,9 +1617,6 @@ static void run_tunneler_loop(uv_loop_t* ziti_loop) {
     }
 
     free(tunneler);
-#if _WIN32
-    close_log();
```

Comment:
this was causing a crash because it nulls the logger file pointer, and logging happens after this point. without crashes, the remainder of `run_tunnel` (after `uv_run` returns) is executed.